Sources/App/Controllers/TodoController.swift

```diff
@@ -20,4 +20,23 @@ final class TodoController {
             return todo.delete(on: req)
         }.transform(to: .ok)
     }
+
+    /// Updates a decoded `Todo` to the database.
+    func update(_ req: Request) throws -> Future<Todo> {
+
```

Comment:
you should use the `router.put(Todo.self, ...)` syntax here :)

Reply:
I don't get the point here.